tools/topology/topology2/include/components/src_format_s32_to_sxx_convert.conf

```diff
@@ -0,0 +1,346 @@
+					out_valid_bit_depth      24
+				}
+				# 44.1khz output
+				{
+					out_rate                 44100
+					out_bit_depth            32
+					out_valid_bit_depth      24
+				}
+				# 48khz output
+				{
+					out_rate                 48000
+					out_bit_depth            32
+					out_valid_bit_depth      24
+				}
+				# 64khz output
+				{
+					out_rate                 64000
+					out_bit_depth            32
+					out_valid_bit_depth      24
+				}
+				# 88.2khz output
+				{
+					out_rate                 88200
+					out_bit_depth            32
+					out_valid_bit_depth      24
+				}
+				# 96khz output
+				{
+					out_rate                 96000
+					out_bit_depth            32
+					out_valid_bit_depth      24
+				}
+				# 176.4khz output
+				{
+					out_rate                 176400
+					out_bit_depth            32
+					out_valid_bit_depth      24
+				}
+				# 192khz output
+				{
+					out_rate                 192000
+					out_bit_depth            32
+					out_valid_bit_depth      24
+				}
+				# 8khz output
+				{
+					out_rate		 8000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 11.025 khz output
+				{
+					out_rate                 11025
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 12khz output
+				{
+					out_rate                 12000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 16khz output
+				{
+					out_rate                 16000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 22.05khz output
+				{
+					out_rate                 22050
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 24khz output
+				{
+					out_rate                 24000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 32khz output
+				{
+					out_rate                 32000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 44.1khz output
+				{
+					out_rate                 44100
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 48khz output
+				{
+					out_rate                 48000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 64khz output
+				{
+					out_rate                 64000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 88.2khz output
+				{
+					out_rate                 88200
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 96khz output
+				{
+					out_rate                 96000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 176.4khz output
+				{
+					out_rate                 176400
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 192khz output
+				{
+					out_rate                 192000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
```

Comment:
do we want to merge this or use the simplifications suggested by @ranj063 with the ArrayObject thingy.

Reply:
I'd proceed with this and change after the object array work is merged. But now there's a conflict again --> need rebase and new check next week.